src/ZTUtils/Zope.py

```diff
@@ -204,8 +205,7 @@ def make_query(*args, **kwargs):
     for i in range(len(qlist)):
         k, m, v = qlist[i]
         if type(v) == unicode:
```

Comment:
Sorry that I've overlooked this one. As in line 296 this should be `if isinstance(v, unicode):`